tests/e2e/pkg/tester/tester.go

```diff
@@ -203,6 +199,22 @@ func (t *Tester) addNodeIG() error {
 			ig = v
 		}
 	}
+	numNodes := int(*ig.Spec.MaxSize) // we assume that MinSize = Maxsize, this is true for e2e testing
+	klog.Infof("Setting -num-nodes=%v", numNodes)
+	t.TestArgs += " -num-nodes=" + strconv.Itoa(numNodes)
+
+	// Set the node-os-distro flag for ginkgo
+	distro := CalculateDistroFromIG(ig)
+	if distro != "" {
+		klog.Infof("Setting --node-os-distro=%s", distro)
+		t.TestArgs += " --node-os-distro=" + distro
```

Comment:
Don't remember, did we say we will move this to the generator for now?

Reply:
Yes

Reply:
This is ready to merged